hastexo/hastexo.py

```diff
@@ -663,7 +663,7 @@ def launch_new_window(self, request, suffix=''):
         """
         The fullscreen lab view, opened in a new browser window.
         """
-        if 'sessionid' not in request.cookies:
+        if not all(c in request.cookies for c in ('sessionid', 'edxloggedin')):
```

Comment:
I had a bit of trouble parsing what this does, so I'd suggest to add a comment to benefit our future selves. :)

```suggestion
        # Properly authenticated requests contain two session cookies: sessionid and edxloggedin.
        # If either of them isn't present, return HTTP 401 immediately.
        if not all(c in request.cookies for c in ('sessionid', 'edxloggedin')):
```

Reply:
Ack, done.

Reply:
Thank you!